main/src/main/java/org/mobilitydata/gtfsvalidator/validator/ShapeIncreasingDistanceValidator.java

```diff
@@ -72,8 +74,15 @@ public void validate(NoticeContainer noticeContainer) {
         }
         // equal shape_dist_traveled and different coordinates
         if (!(curr.shapePtLon() == prev.shapePtLon() && curr.shapePtLat() == prev.shapePtLat())) {
-          noticeContainer.addValidationNotice(
-              new EqualShapeDistanceDiffCoordinatesNotice(prev, curr));
+          double distanceBetweenShapePoints =
+              getDistanceMeters(curr.shapePtLatLon(), prev.shapePtLatLon());
+          if (distanceBetweenShapePoints >= DISTANCE_THRESHOLD) {
+            noticeContainer.addValidationNotice(
+                new EqualShapeDistanceDiffCoordinatesNotice(prev, curr));
+          } else if (distanceBetweenShapePoints > 0) {
+            noticeContainer.addValidationNotice(
+                new EqualShapeDistanceDiffCoordinatesWarningNotice(prev, curr));
```

Comment:
Update warning notice name to: `equal_shape_distance_diff_coordinates_distance_below_1.1m`

Reply:
i cannot put a `.` in the name of the notice